examples/deploy/README.md

```diff
@@ -0,0 +1,14 @@
+### Example production deployment of bokeh server.
+
+Execute each of the following in separate shells
```

Comment:
If you're working with deployment, I would add [honcho][] here to management running everything and change this to simply be `honcho start`.  This is the pattern we started implementing in the various Wakari projects and should really be used throughout.

[honcho]: https://github.com/nickstenning/honcho